From 15f21f522cb778c2971adcefb93936fb8aeb2a4c Mon Sep 17 00:00:00 2001
From: WangJing <wangjing@mprc.pku.edu.cn>
Date: Wed, 28 Sep 2011 17:28:59 +0800
Subject: [PATCH 005/641] UniCore64: Add user.h

Fixed error:include/linux/user.h:1:22: fatal error: asm/user.h: No such file or directory

Signed-off-by: WangJing <wangjing@mprc.pku.edu.cn>
---
 arch/unicore64/include/asm/Kbuild |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/unicore64/include/asm/Kbuild b/arch/unicore64/include/asm/Kbuild
index c03e36e..fdba13e 100644
--- a/arch/unicore64/include/asm/Kbuild
+++ b/arch/unicore64/include/asm/Kbuild
@@ -63,3 +63,4 @@ generic-y += topology.h
 generic-y += types.h
 generic-y += uaccess.h
 generic-y += unaligned.h
+generic-y += user.h
-- 
1.7.9.5

